.github/ISSUE_TEMPLATE/bug-report-or-feature-request.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report or feature request
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Provide a general summary of the issue in the Title above.
+
+**Expected Behavior**
+If you're describing a bug, tell us what should happen. If you're suggesting a change/improvement, tell us how it should work.
+
+**Current Behavior**
+If describing a bug, tell us what happens instead of the expected behavior.
+Include full errors, uncaught exceptions, stack traces, and relevant logs.
+If service responses are relevant, please include Fiddler-captured wirelogs.
+If suggesting a change/improvement, explain the difference from current behavior.
+
+**Possible Solution***
```

Comment:
it looks like i missed it the first time.  this one has extra *

Reply:
Done